packages/salesforcedx-vscode-apex-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -970,10 +974,10 @@ export async function sfdxToggleCheckpoint() {
   }
   const bpAdd: vscode.Breakpoint[] = [];
   const bpRemove: vscode.Breakpoint[] = [];
-  const uri = exports.fetchActiveEditorUri();
-  const lineNumber = exports.fetchActiveSelectionLineNumber();
+  const uri = checkpointUtils.fetchActiveEditorUri();
+  const lineNumber = checkpointUtils.fetchActiveSelectionLineNumber();
 
-  if (uri && lineNumber) {
+  if (uri && lineNumber !== undefined) {
```

Comment:
when debugging tests found that we have a flow where lineNumber is 0 ....which is falsey here 😢